docs/intro/installation.md

```diff
@@ -0,0 +1,354 @@
+---
+title: Installation
+sidebarSortOrder: 1
+---
```

Comment:
since this content is largely being ported from the guide, lets remove the guide doc too and add in the description (since it will show on social card and search engines)

```suggestion
---
title: Installation
sidebarSortOrder: 1
description:
  "Easily setup your local development environment for Solana development on
  Linux, Mac or Windows. Including installing Rust, the Solana CLI, and Anchor."
altRoutes:
  - /guides/getstarted/setup-local-development
---
```